easybuild/easyconfigs/b/BLASR/BLASR-20170330-intel-2017a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'MakeCp'
+
+name = 'BLASR'
+commit = 'bed926d'
+version = '20170330'
+
+homepage = 'https://github.com/PacificBiosciences/blasr'
+description = """ BLASR (Basic Local Alignment with Successive Refinement) rapidly maps 
+ reads to genomes by finding the highest scoring local alignment or set of local alignments 
+ between the read and the genome. Optimized for PacBio's extraordinarily long reads and 
+ taking advantage of rich quality values, BLASR maps reads rapidly with high accuracy. """
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+#toolchainopts = {'usempi': True}
```

Comment:
I got MPI-related errors when linking with a non-serial HDF5, and this was an earlier attempt to fix it, I'll get rid of it, thanks.